src/es/ikigaimangas/src/eu/kanade/tachiyomi/extension/es/ikigaimangas/IkigaiMangas.kt

```diff
@@ -33,7 +33,7 @@ import kotlin.concurrent.thread
 
 class IkigaiMangas : HttpSource(), ConfigurableSource {
 
-    override val baseUrl = "https://ikigaimangas.com"
+    override val baseUrl = "https://lectorikigai.acamu.net"
 
     private val defaultBaseUrl: String = "https://lectorikigai.acamu.net"
 
```

Comment:
baseUrl and defaultBaseUrl are the same

Reply:
defaultBaseUrl is used as fallback when baseUrl doesn't match the domain detected by the extension.... Since there's nothing else to fallback to (at least to my knowledge), I've left them the same.

Additionally, this extension by default enables automatically detecting domain url for api calls, which would override both of these if that preference is enabled.

Reply:
This will break the method for automatically fetching the domain. You can change it to:
```kotlin
  private val isCi = System.getenv("CI") == "true"
  override val baseUrl get() = when {
      isCi -> defaultBaseUrl
      else -> preferences.getPrefBaseUrl()
  }
```
as a previous version and hardcode de main domain in the method